docs/2.guide/2.directory-structure/1.server.md

```diff
@@ -272,7 +272,7 @@ export default fromNodeMiddleware((req, res) => {
 ```
 
 ::alert{type=warning}
-Legacy support is possible using [unjs/h3](https://github.com/unjs/h3), but it is advised to avoid legacy handlers as much as you can.
+Legacy support is possible using [unjs/h3](https://github.com/unjs/h3), though it's advised to avoid legacy handlers as much as possible.
```

Comment:
```suggestion
Legacy support is possible using [H3](https://github.com/unjs/h3), though it's advised to avoid legacy handlers as much as possible.
```